vg/vg.go

```diff
@@ -85,6 +88,49 @@ type CanvasSizer interface {
 	Size() (x, y Length)
 }
 
+// CanvasWriterTo wraps behavior required for writing
+// a Canvas representation to a concrete image.
+type CanvasWriterTo interface {
+	CanvasSizer
+	io.WriterTo
+}
+
+var (
+	formatLock sync.RWMutex
+	formats    = make(map[string]func(w, h Length) CanvasWriterTo)
```

Comment:
wrt the title: both openGL and X11 require a title for the new window in which the canvas will live.
as for EPS, an empty string may do, but it feels odd.
perhaps as odd as giving an empty string to PDF, PNG, ... factory functions, though.

if only one could connect the Title of the `plot.Plot` to the canvas creation...